packs/spell-effects/spell-effect-dragon-wings-10-minutes.json

```diff
@@ -0,0 +1,100 @@
+{
+    "_id": "V7jAnItnVqtfCAKt",
+    "img": "icons/creatures/abilities/wing-batlike-red-pink.webp",
+    "name": "Spell Effect: Dragon Wings (10 minutes)",
+    "system": {
+        "description": {
+            "value": "<p>Granted by casting @UUID[Compendium.pf2e.spells-srd.Item.Dragon Wings].<p>"
+        },
+        "duration": {
+            "expiry": "turn-start",
+            "sustained": false,
+            "unit": "minutes",
+            "value": 10
+        },
+        "level": {
+            "value": 5
+        },
+        "publication": {
+            "license": "OGL",
+            "remaster": false,
+            "title": "Pathfinder Core Rulebook"
+        },
+        "rules": [
+            {
+                "key": "BaseSpeed",
+                "selector": "fly",
+                "value": "max(60, @actor.attributes.speed.total)"
+            },
+            {
+                "key": "ChoiceSet",
+                "adjustName": true,
+                "choices": [
+                    {
+                        "label": "1 Focus Point",
+                        "value": 1
+                    },
+                    {
+                        "label": "2 Focus Points",
+                        "value": 2
+                    }
+                ],
+                "prompt": "Focus Points spent",
+                "rollOption": "dragon-wings-focus-points"
+            },
```

Comment:
Drop this and instead put an ItemAlteration on the spell and a toggle placed on the spellcasting tab. Then a note that will add a link to the dragon claws effect at the same time. see #13284 for how to do those things.